media-libs/libjxl/libjxl-0.9.1-r1.ebuild

```diff
@@ -42,6 +42,7 @@ RDEPEND="${DEPEND}"
 PATCHES=(
 	"${FILESDIR}/${PN}-0.9.1-backport-pr3138.patch"
 	"${FILESDIR}/${PN}-0.9.1-backport-pr3143.patch"
+	"${FILESDIR}/${PN}-0.9.1-backport-pr3218.patch"
```

Comment:
That's quite a big patch. Can't we stabilize 0.9.2 instead?

Reply:
_Hello @mgorny,_

It would be better, sure thing. Sadly, I have no idea how long it usually takes to stabilize a new minor version.

I know your are aware of it, but for others, there is a PR that brings 0.9.2:
  - #35827 

I can drop the patch from this PR and keep only a fix for `multilib_foreach_abi gnome2_gdk_pixbuf_update`, but I'm not quite sure how bad [the corresponding issue](https://github.com/libjxl/libjxl/issues/3144) is.

Reply:
I've diffed it and I think it's fine to stabilize it as soon as I push it. So yeah, let's limit the change to fixing postinst/postrm.

Reply:
Agreed. I've updated the PR.

Reply:
_For transparency, here is the corresponding [stable request](https://wiki.gentoo.org/wiki/Stable_request):_
  - [media-libs/libjxl-0.9.2: stablereq](https://bugs.gentoo.org/927307)